include/glow/Importer/ProtobufLoader.h

```diff
@@ -142,14 +144,14 @@ class ProtobufLoader {
   /// classification, this single final output is usually the result of the last
   /// softmax or regression layer.
   /// \pre outputVarsByName_.size() == 1
-  Variable *getSingleOutput() {
+  Placeholder *getSingleOutput() {
     assert(outputVarsByName_.size() == 1);
     return outputVarsByName_.begin()->second;
   }
 
   /// \returns the Variable for the external output with \p name.
   /// \pre outputVarsByName_.find(name) != outputVarsByName_.end()
-  Variable *getOutputByName(llvm::StringRef name) const;
+  Storage *getOutputByName(llvm::StringRef name) const;
```

Comment:
Placeholder as well?

Reply:
No, because the last input could be a constant. At least I think that this is possible.

Reply:
> No, because the last input could be a constant

hm, this is getOutputByName. What do you mean by last input?

Reply:
Ah, you are correct. Sorry. Yes, I will fix this.

Reply:
It seems that output is taken from outputVarsByName_, which has Placeholders.